hbase/src/test/java/org/apache/metamodel/hbase/HBaseUpdateCallbackTest.java

```diff
@@ -0,0 +1,283 @@
+     */
+    protected void dropTableIfItExists() {
+        if (schema != null) {
+            final Table table = schema.getTableByName(TABLE_NAME);
+            if (table != null) {
+                updateCallback.dropTable(table).execute();
+                // Check schema
+                assertNull(schema.getTableByName(TABLE_NAME));
+                // Check in the datastore
+                try (final Admin admin = getDataContext().getAdmin()) {
+                    assertFalse(admin.tableExists(TableName.valueOf(TABLE_NAME)));
+                } catch (IOException e) {
+                    fail("Should not an exception checking if the table exists");
+                }
+            }
+        }
+    }
+
+    /**
+     * Check if the table has been inserted succesfully.
+     * Checks are performed in the schema and the datastore.
+     * @throws IOException because the admin object needs to be created
+     */
+    protected void checkSuccesfullyInsertedTable() throws IOException {
+        // Check the schema
+        assertNotNull(schema.getTableByName(TABLE_NAME));
+        // Check in the datastore
+        try (final Admin admin = getDataContext().getAdmin()) {
+            assertTrue(admin.tableExists(TableName.valueOf(TABLE_NAME)));
+        } catch (IOException e) {
+            fail("Should not an exception checking if the table exists");
+        }
+    }
+
+    /**
+     * Create a test HBase table and add it to the datastore
+     * @param tableName name of the table
+     * @param idColumn required ID-column
+     * @param columnFamily1 required columnFamily 1
+     * @param columnFamily2 required columnFamily 2
+     * @return created and add {@link HBaseTable}
+     * @throws IOException
+     */
+    protected HBaseTable createAndAddTableToDatastore(final String tableName, final String idColumn,
+            final String columnFamily1, final String columnFamily2) throws IOException {
+        final LinkedHashSet<String> columnFamilies = new LinkedHashSet<>();
+        columnFamilies.add(idColumn);
+        columnFamilies.add(columnFamily1);
+        columnFamilies.add(columnFamily2);
+        updateCallback.createTable(schema, tableName, columnFamilies).execute();
+        checkSuccesfullyInsertedTable();
+        return (HBaseTable) getDataContext().getDefaultSchema().getTableByName(tableName);
+    }
+
+    /**
+     * Create a test HBase table
+     * @param tableName name of the table
+     * @param idColumn ID-column, can be set to null to create a table without this column
+     * @param columnFamily1 required columnFamily 1
+     * @param columnFamily2 required columnFamily 2
+     * @param columnFamily3 columnFamily 3 is not required and can be used to test errors
+     * @return created {@link HBaseTable}
+     */
+    protected HBaseTable createHBaseTable(final String tableName, final String idColumn, final String columnFamily1,
+            final String columnFamily2, final String columnFamily3) {
```

Comment:
I don't like this, can't you use one columnFamily parameter instead which uses the Varargs feature, so `final String... columnFamilies`?